scripts/assisted_deployment.sh

```diff
@@ -5,6 +5,11 @@ function destroy_all() {
 }
 
 function set_dns() {
```

Comment:
will the dns be deleted on destroy all? I mean if we deploy once with NM configuration but then want to deploy without it?
@tsorya @YuviGold